.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,41 @@
-## Proposed Changes
+Describe proposed changes and the issue being fixed
 
-  -
-  -
-  -
+<!-- Optional -->
+Fixes [link to the issue]
 
 ## Testing
+<!-- Optional -->
+Describe how you tested your changes. If possible and needed:
+- Test it on a sample project
+- Write unit tests
+- Provide a code snippet
 
-Test: Describe how you tested your changes. Note that this line (with `Test:`) is required, your PR will not build without it!
+<!-- Optional -->
+This should be tested by QA
 
-## Issues Fixed
+## Release Notes
+<!--
+Optional, if omitted - won't be included in the changelog
 
-Fixes: [Optional] The bug on [https://issuetracker.google.com](https://issuetracker.google.com) being fixed
+Sections:
+- Highlights
+- Known issues
+- Breaking changes
+- Features
+- Fixes
+- Prerelease fixes
+> Prerelease fixes are fixes of bugs introduced in a previous prerelease version (dev/beta). It will be excluded from the stable changelog
+
+Subsections:
+- Multiple Platforms
```

Comment:
So if a fix was for k/native and k/js,k/wasm. Do we choose "Multiple Platforms", despite the fix is not applicable for desktop?
___

Another note:
"Resources" and "Gradle Plugin" is a bit different category than platforms. They are rather a components. 
Fixes in "Resources" and "Gradle plugin" can be platform specific as well.

But for simplicity we can keep it like you propose here. If it's not enough, we can add one more "category" later.

Reply:
> Do we choose "Multiple Platforms", despite the fix is not applicable for desktop?
Yes, it is to avoid adding an additional dimension - it can complicate things.

> They are rather "components".
> Fixes in "Resources" and "Gradle plugin" can be platform specific as well.

It is more about subsystems rather than targets and components.

We treat "Multiple Platforms", "Desktop", "Web", "iOS" as "Compose. Multiple Platforms", "Compose. Desktop", ... But we omit "Compose" as it is the main changelog for it.